tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -906,7 +904,8 @@ TEST (nnstreamer_nnfw_mlapi, invoke_pipeline_01_n)
       ml_pipeline_src_input_data (src_handle, input,
       ML_PIPELINE_BUF_POLICY_DO_NOT_FREE);
   EXPECT_EQ (status, ML_ERROR_INVALID_PARAMETER);
-  g_usleep (100000);
+
+  g_usleep (500000); /* wait for sink callback */
```

Comment:
previous line (to push data into src element) returns invalid-param, and we dont need to wait sink event here.
(no callback registered in this tc)

Reply:
I agree, it's removed